test/sensitivities/functional/functional.jl

```diff
@@ -1,9 +1,14 @@
 using SpecialFunctions
 using DiffRules: diffrule, hasdiffrule
 
+# ones(::AbstractArray) is deprecated in 0.7 and removed in 1.0, but it's a pretty useful
+# method, so we'll define our own for testing purposes
+oneslike(a::AbstractArray) = ones(eltype(a), size(a))
+oneslike(n::Integer) = ones(n)
```

Comment:
Do we want to have this inside of Nabla to avoid having `zeros(eltype(x), size(x))`?